test/fulfillmentSpec.js

```diff
@@ -332,162 +326,4 @@ describe('PUT /fulfillment', function () {
     expect((yield getAccount('alice')).balance).to.equal(100)
     expect((yield getAccount('bob')).balance).to.equal(0)
   })
-
-  it('should trigger subscriptions when notification is executed', function * () {
```

Comment:
Tests for notification when transfers are executed / rejected can be found in https://github.com/interledger/five-bells-ledger/blob/ws_notifications/test/accountSpec.js#L771 and https://github.com/interledger/five-bells-ledger/blob/ws_notifications/test/accountSpec.js#L825

Not sure why they were put in `accountSpec.js` in the first place.